### PR TITLE
Add billing docs

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Added billing docs to grid client

### Related Issues

- #3080
### Documentation PR

For UI changes, Please provide the Documetation PR on [info_grid](https://github.com/threefoldtech/info_grid)

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/grid_client/docs/billing.md [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+
+### Applying the discounts
+
+Before assuming that the price above is the final price, check first if your twin is applicable for any of the available staking discount levels. to understand more about discount levels please check the [manual](https://manual.grid.tf/cloud/cloudunits_pricing.html#staking-discount).
+
+For this example lets assume that this twin has 18 months worth of TFTs staked, so the user will be applicable for a Gold discount level (60% discount).
+
+```sh
+Cost with 60% discount = 0.943180875 * 0.4
+                       = 0.37727235000000003 TFT/hour
```
Comment (amiraabouhadid) [inline packages/grid_client/docs/billing.md]: please use only 2 or 3 decimal places

**Review by zaelgohary — CHANGES_REQUESTED** [review]
I think this document should be linked to the dedicated nodes when a user reserves a node. Additionally, it should be connected to the node contracts page and the calculation information for each solution.

**Comment by zaelgohary** [comment]
Also, this should be in Playground docs not the Grid client.

File: packages/grid_client/docs/billing.md [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+
+A couple of examples explaining how the billing works on the threefold grid for node and rent contracts.
+
+Some of the used resources should be updated whenever you try to do these calculations, these resources are the TFT price and Cloud Units(SU, CU).
+
+## Current TFT Price
+
+TFT price can be retrieved either from the dashboard, through the price available in the header
+
+![image](https://user-images.githubusercontent.com/101194226/252134103-e7cce566-311d-4293-8e73-358d45b616d1.png)
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]: Please update the image.

image

File: packages/grid_client/docs/billing.md [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+```sh
+Cru: 2
+Mru: 2
+Sru: 15
+Hru: 0
+```
+
+### Calculating the CU
+
+The current CU value is `10 mUSD/h`, make sure that this value is updated according to the details section.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]: What does this mean?

File: packages/grid_client/docs/billing.md [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+   = min( 1, 2, 1 )
+   = 1
+CU cost/hour = CU * CU pricing
+             = 1 * 10 mUSD/h
+             = 10 mUSD/h
+```
+
+### Calculating the SU
+
+The current SU value is `5 mUSD/h`, make sure that this value is updated according to the details section.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]: here too.

File: packages/grid_client/docs/billing.md [hunk 5]
```diff
@@ -0,0 +1,334 @@
...
+   = 0 + 0.075
+   = 0.075
+SU cost/hour = SU * SU pricing
+             = 0.075 * 5 mUSD/h
+             = 0.375 mUSD/h
+```
+
+### Calculating the Billing Rate for the contract
+
+The current TFT value is `0.011 USD`, make sure that this value is updated according to the details section.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]: and here.

File: packages/grid_client/docs/billing.md [hunk 6]
```diff
@@ -0,0 +1,334 @@
...
+                   = 0.010375 USD/h
+                   = (0.010375 * 24 * 30)
+                   = 7.47 USD/month
+                   = 679.09023 TFT/month
+                   = 0.943180875 TFT/hour
+```
+
+### Applying the discounts
+
+Before assuming that the price above is the final price, check first if your twin is applicable for any of the available staking discount levels. to understand more about discount levels please check the [manual](https://manual.grid.tf/cloud/cloudunits_pricing.html#staking-discount).
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]: Can we rephrase this?

File: packages/grid_client/docs/billing.md [hunk 7]
```diff
@@ -0,0 +1,334 @@
...
+```sh
+CRU = 4
+SRU = 119.24
+HRU = 1863
+MRU = 15.55
+```
+
+### Calculating CU
+
+The current CU value is `10 mUSD/h`, make sure that this value is updated according to the details section.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]: here too.

File: packages/grid_client/docs/billing.md [hunk 8]
```diff
@@ -0,0 +1,334 @@
...
+   = min( 3.8875, 4, 7.775 )
+   = 3.8875
+CU cost/hour = CU * CU pricing
+             = 3.8875 * 10 mUSD/h
+             = 38.875 mUSD/h
+```
+
+### Calculating SU
+
+The current SU value is `5 mUSD/h`, make sure that this value is updated according to the details section.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]: and here.

File: packages/grid_client/docs/billing.md [hunk 9]
```diff
@@ -0,0 +1,334 @@
...
+There's a default `50%` discount for renting a node, this discount is not related to the staking discount. for more info please check the [manual](https://manual.grid.tf/cloud/cloudunits_pricing.html#dedicated-nodes).
+
+```sh
+Cost with 50% discount = 35.725319999999996 * 0.5
+                       = 17.862659999999998 TFT/month
+```
+
+### Applying the Staking discount
+
+Before assuming that the price above is the final price, check first if your twin is applicable for any of the available staking discount levels. to understand more about discount levels please check the [manual](https://manual.grid.tf/cloud/cloudunits_pricing.html#staking-discount).
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]: Please rephrase this.

File: packages/grid_client/docs/billing.md [hunk 10]
```diff
@@ -0,0 +1,334 @@
...
+And since the current TFT conversion rate is `1 USD = 100 TFT`
+
+```sh
+uniqueName in TFT = 0.00025 * 100
+                  = 0.025 TFT/hour
+```
+
+### Applying Staking discount
+
+Before assuming that the price above is the final price, check first if your twin is applicable for any of the available staking discount levels. to understand more about discount levels please check the [manual](https://manual.grid.tf/cloud/cloudunits_pricing.html#staking-discount).
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]: Rephrase this too.

**Review by zaelgohary — CHANGES_REQUESTED** [review]
While staking discounts are the same in all contracts, it's being repeated 3 times with dedicated nodes, node w public IP & with NU. Can we find a way to reduce the repetition?

**Comment by maayarosama** [comment]
> I think this document should be linked to the dedicated nodes when a user reserves a node. Additionally, it should be connected to the node contracts page and the calculation information for each solution.

Per discussion with @AhmedHanafy725 this's not needed, as users can easily check the prices using the calculator

File: packages/grid_client/docs/billing.md [hunk 11]
```diff
@@ -0,0 +1,334 @@
...
+
+TFT price can be retrieved either from the dashboard, through the price available in the header
+
+![image](./assets/dashboard.png)
+
+or directly through [stellar](https://stellar.expert/explorer/public/asset/TFT-GBOVQKJYHXRR3DX6NOX2RRYFRCUMSADGDESTDNBDS6CDVLGVESRTAC47-1?asset[]=TFT-GBOVQKJYHXRR3DX6NOX2RRYFRCUMSADGDESTDNBDS6CDVLGVESRTAC47-1&filter=markets&market=USDC-GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN-1).
+
+## Current Cloud Units value
+
+These can be retrieved directly from the chain. Through the [polakdot UI](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftfchain.grid.tf#/chainstate).
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]:
```suggestion
These can be retrieved directly from the chain. Through the [Polakdot UI](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftfchain.grid.tf#/chainstate).
```

File: packages/grid_client/docs/billing.md [hunk 12]
```diff
@@ -0,0 +1,334 @@
...
+
+Enter the value of the default pricing policy which is `1` or enter the value of any other policy if you need to use a custom one.
+
+![image](https://user-images.githubusercontent.com/101194226/252134747-8e4da69a-b4c9-4a1b-bc93-82398f95d6c9.png)
+
+> Values on chain are expressed as "units USD per hour" where "1 unit usd" == 10.000.000 (1e7)
+
+## Node Contract
+
+Lets assume that the resources for this deployment are:
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]:
```suggestion
Let's assume that the resources for this deployment are:
```

File: packages/grid_client/docs/billing.md [hunk 13]
```diff
@@ -0,0 +1,334 @@
...
+Cost with 60% discount = 0.4 * 0.4
+                       = 0.160 TFT/hour
+
+```
+
+> Note: this value gets added to the billing rate of your deployment.
+
+## Network Usage
+
+Network Usage is calculated for deployments with public ips, it's reprted every hour and its cost can be calculated approximatly as the value of data(sent + recieved) \* the NU value.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]:
```suggestion
Network Usage is calculated for deployments with public IPs, it's reported every hour and its cost can be calculated approximately as the value of data(sent + received) \* the NU value.
```

File: packages/grid_client/docs/billing.md [hunk 14]
```diff
@@ -0,0 +1,334 @@
...
+
+So firstly lets calculate the data usage. This can be tracked with network tool like `nload` where it can show the total of the sent and recived.
+
+![image](https://user-images.githubusercontent.com/101194226/253294544-d4691e9a-84af-4f85-a6cc-6cd3f3902448.png)
+
+After than we can get the NU value. you can get its current value from the Pricing Policy.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]:
```suggestion
After then we can get the NU value. you can get its current value from the Pricing Policy.
```

File: packages/grid_client/docs/billing.md [hunk 15]
```diff
@@ -0,0 +1,334 @@
...
+
+```
+
+As an example lets assume that we used a total of 10GB in the last hour, so the next hour the billing rate should be updated to:
+
+```sh
+Total network usage = 10GB * 0.06 TFT/hour
+                    = 0.6 TFT/hour
+
+Billing rate in the next hour should be: actual cost of the deployment + Total Netowrk Usage.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]:
```suggestion
The billing rate in the next hour should be: the actual cost of the deployment + Total Network Usage.
```

File: packages/grid_client/docs/billing.md [hunk 16]
```diff
@@ -0,0 +1,334 @@
...
+
+Some of the used resources should be updated whenever you try to do these calculations, these resources are the TFT price and Cloud Units(SU, CU).
+
+## Current TFT Price
+
+TFT price can be retrieved either from the dashboard, through the price available in the header
+
+![image](./assets/dashboard.png)
+
+or directly through [stellar](https://stellar.expert/explorer/public/asset/TFT-GBOVQKJYHXRR3DX6NOX2RRYFRCUMSADGDESTDNBDS6CDVLGVESRTAC47-1?asset[]=TFT-GBOVQKJYHXRR3DX6NOX2RRYFRCUMSADGDESTDNBDS6CDVLGVESRTAC47-1&filter=markets&market=USDC-GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN-1).
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]:
```suggestion
or directly through [Stellar](https://stellar.expert/explorer/public/asset/TFT-GBOVQKJYHXRR3DX6NOX2RRYFRCUMSADGDESTDNBDS6CDVLGVESRTAC47-1?asset[]=TFT-GBOVQKJYHXRR3DX6NOX2RRYFRCUMSADGDESTDNBDS6CDVLGVESRTAC47-1&filter=markets&market=USDC-GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN-1).
```

File: packages/grid_client/docs/billing.md [hunk 17]
```diff
@@ -0,0 +1,334 @@
...
+   = 0 + 0.075
+   = 0.075
+SU cost/hour = SU * SU pricing
+             = 0.075 * 5 mUSD/h
+             = 0.375 mUSD/h
+```
+
+### Calculating the Billing Rate for the contract
+
+The current TFT value is `0.012 USD`, please note that prices could be subjected to change.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]:
```suggestion
The current TFT value is `0.012 USD`. Please note that prices could be subjected to change.
```

File: packages/grid_client/docs/billing.md [hunk 18]
```diff
@@ -0,0 +1,334 @@
...
+   = min( 1, 2, 1 )
+   = 1
+CU cost/hour = CU * CU pricing
+             = 1 * 10 mUSD/h
+             = 10 mUSD/h
+```
+
+### Calculating the SU
+
+The current SU value is `5 mUSD/h`, please note that prices could be subjected to change.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md:56]:
```suggestion
The current SU value is `5 mUSD/h`. Please note that prices could be subjected to change.
```

File: packages/grid_client/docs/billing.md [hunk 19]
```diff
@@ -0,0 +1,334 @@
...
+```sh
+CRU = 4
+SRU = 119.24
+HRU = 1863
+MRU = 15.55
+```
+
+### Calculating CU
+
+The current CU value is `10 mUSD/h`, please note that prices could be subjected to change.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]:
```suggestion
The current CU value is `10 mUSD/h`. Please note that prices could be subjected to change.
```

File: packages/grid_client/docs/billing.md [hunk 20]
```diff
@@ -0,0 +1,334 @@
...
+   = min( 3.8875, 4, 7.775 )
+   = 3.8875
+CU cost/hour = CU * CU pricing
+             = 3.8875 * 10 mUSD/h
+             = 38.875 mUSD/h
+```
+
+### Calculating SU
+
+The current SU value is `5 mUSD/h`, please note that prices could be subjected to change.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]:
```suggestion
The current SU value is `5 mUSD/h`. Please note that prices could be subjected to change.
```

File: packages/grid_client/docs/billing.md [hunk 21]
```diff
@@ -0,0 +1,334 @@
...
+   = 1.5525 + 0.5962
+   = 2.1487
+SU cost/hour = SU * SU pricing
+             = 2.1487 * 5 mUSD/h
+             = 10.7435 mUSD/h
+```
+
+### Calculating Billing Rate for the contract
+
+The current TFT value is `0.012 USD`, please note that prices could be subjected to change.
```
Comment (zaelgohary) [inline packages/grid_client/docs/billing.md]:
```suggestion
The current TFT value is `0.012 USD`. Please note that prices could be subjected to change.
```

**Review by zaelgohary — APPROVED** [review]

**Review by amiraabouhadid — APPROVED** [review]
